core/shared/src/main/scala-3/org/log4s/LoggerMacros.scala

```diff
@@ -54,32 +54,32 @@ private[log4s] object LoggerMacros {
       '{ new Logger(JLoggerFactory.getLogger($name)) }
     }
 
-    val cls = findEnclosingClass(Symbol.currentOwner)
+    val cls = findEnclosingClass(Symbol.spliceOwner)
```

Comment:
There is no [Symbol.currentOwner](https://github.com/lampepfl/dotty/pull/10394/files?diff=split&w=1#diff-f908fb3c43c3907a65be1e130486551c494bddc402c769a5ffcb055c74b0cb04L2206) anymore.
I think [Symbol.spliceOwner](https://github.com/lampepfl/dotty/blob/3.0.0-M2/compiler/src/scala/quoted/runtime/impl/QuotesImpl.scala#L2207-L2208) is equivalent to the old `Symbol.currentOwner`.